src/Persistence/MassTransit.MongoDbIntegration/readme.md

```diff
@@ -0,0 +1,216 @@
+# MassTransit.MongoDbIntegration
+
+This package supports MongoDb, including the storage of sagas, message data, and routing slip events in MongoDB. For sagas, this means that you can orchestrate a long-running process and manage its state throughout execution. For message data, you can use GridFS to store large files. For routing slips, you can store the events produced by a routing slip as documents.
+
+## Attribution
+
+Many parts of this package, including the message data and saga repositories, were originally created by Liberis Labs, including [Lee Blundell](https://github.com/Blundell89), [Kevin Smith](https://github.com/kevbite), and others. The original packages were [here](https://github.com/LiberisLabs/MassTransit.MessageData.MongoDb) and [here](https://github.com/LiberisLabs/MassTransit.Persistence.MongoDb). They were copied with permission from the authors into this official assembly.

```

Comment:
Please can we add the following link to `Liberis Labs`? http://goo.gl/VaFgpk

Thanks!